test/unit/dimensions.js

```diff
@@ -757,7 +757,11 @@ QUnit.test( "outerWidth/Height for table cells and textarea with border-box in I
 		$firstTh = jQuery( "<th style='width: 200px;padding: 5px' />" ),
 		$secondTh = jQuery( "<th style='width: 190px;padding: 5px' />" ),
 		$thirdTh = jQuery( "<th style='width: 180px;padding: 5px' />" ),
-		$td = jQuery( "<td style='height: 20px;padding: 5px;border: 1px solid'>text</td>" ),
+		// Support Firefox 63, Edge 16-17, Android 8, iOS 7-11
```

Comment:
A colon is missing after `Support`. :) I'm sometimes searching for `Support:` to see if something's redundant now, this line would be missed then.